sklearn/tests/test_public_functions.py

```diff
@@ -164,6 +164,7 @@ def _check_function_param_validation(
     "sklearn.metrics.cluster.contingency_matrix",
     "sklearn.metrics.cohen_kappa_score",
     "sklearn.metrics.confusion_matrix",
+    "sklearn.metrics.pairwise.cosine_similarity",
```

Comment:
Please move it to preserve alphabetical order

Reply:
Will do